documentation/assertions/spy/was-called-on.md

```diff
@@ -40,12 +41,11 @@ expect(obj.spy, 'was always called on', obj);
 
 ```output
 expected spy was always called on { spy: spy }
-  expected invocations( spy(), spy() )
-  to have values satisfying 'was called on', { spy: spy }
+  expected spyCalls to have values satisfying 'was called on', { spy: spy }
 
   invocations(
-    spy(),
-    spy() // expected spy() was called on { spy: spy }
-          //   expected spy to be called with { spy: spy } as this but was called with {}
+    spy() at theFunction (theFileName:xx:yy),
+    spy() at theFunction (theFileName:xx:yy) // expected spy() at theFunction (theFileName:xx:yy) was called on { spy: spy }
+                                             //   expected spy to be called with { spy: spy } as this but was called with {}
   )
```

Comment:
Maybe https://github.com/unexpectedjs/unexpected/issues/175 could help out with omitting the stack trace line in the error message on the right.

Or we could do a generic fix for `to satisfy` diffs so that the annotation block is moved to the next line and not indented so much when it and the thing it's annotating are wide.